tools/polymer/example/polybench/split-and-merge/correlation/correlation.heuristic.polymer.par.mlir

```diff
@@ -0,0 +1,417 @@
+#map0 = affine_map<(d0) -> (d0)>
```

Comment:
What is the purpose of this example directory? I see .c and .h files, which may remain, but I would remove the log files and the mlir ones.